Makefile

```diff
@@ -139,6 +139,20 @@ rm -rf $$TMP_DIR ;\
 }
 endef
 
+DEPLOYMENT_DIR = $(PROJECT_DIR)/config/deploy/$(OPERATOR_VERSION)
+DEPLOYMENT_TMP = $(PROJECT_DIR)/config/deploy/tmp
+.PHONY: deploy-manifest
+deploy-manifest: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
+	mkdir -p $(DEPLOYMENT_DIR) && mkdir -p $(DEPLOYMENT_TMP)
+	curl $(AUTHORINO_MANIFESTS) > $(DEPLOYMENT_TMP)/authorino-manifests.yaml
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMAGE) ;\
+	cd $(PROJECT_DIR) && $(KUSTOMIZE) build config/default > $(DEPLOYMENT_TMP)/authorino-operator.yaml
+	$(KUSTOMIZE) build config/deploy > $(DEPLOYMENT_DIR)/manifests.yaml
```

Comment:
We are here using kustomize only to merge the two manifest files into one single multi-document YAML. I was wondering whether you were planning on adding more kustomize configs in the future. If not, maybe we could keep it simple with sth like:

```make
deploy-manifest: manifests kustomize
	wget -q -c -P $(DEPLOYMENT_DIR) $(AUTHORINO_MANIFESTS) && echo '---' >> $(DEPLOYMENT_DIR)/$$(basename $(AUTHORINO_MANIFESTS))
	cd $(PROJECT_DIR)/config/manager && $(KUSTOMIZE) edit set image controller=$(OPERATOR_IMAGE)
	cd $(PROJECT_DIR) && $(KUSTOMIZE) build config/default >> $(DEPLOYMENT_DIR)/$$(basename $(AUTHORINO_MANIFESTS))
	cd $(PROJECT_DIR)/config/manager && $(KUSTOMIZE) edit set image controller=$(DEFAULT_OPERATOR_IMAGE)
```

Reply:
Nice one, wI didn't think about adding the `echo '---' ` to the end of the authorino manifest